### ISSUE TITLE
downdraft table/"sanding box"

### ISSUE THREAD

**Issue by LukasFreeze** [comakingspace/do-something#169]
After the move I am finally going to finish the sanding box (found materials forbthat over a year ago, but never got around to building it because it was so easy to just sand outside 🙈).

It will just be a simple wooden box with a shop vac port in one wall and perforated sheet metal at the top, but there is still time to suggest features! @FHuchinson and @NitramLegov were concerned about working height yesterday, so it will be made high enough to have "standing workbench comfort" on top of the relatively low sanding corner.

**Comment by LukasFreeze** [comment]
aus diversen Gründen fehlte mir hierfür doch die Zeit und ich hatte auch keine Schleifprojekte, ich finde es aber immer noch sinnvoll so eine Box zu bauen. Sollte bei der jetzt anstehenden Aufräumaktion der Schleifecke das Lochblech "rausfliegen", legt es mir also bitte hin und entsorgt es nicht :-)

Oder es baut einfach jemand die Kiste, falls viele Leute übermorgen zum "do-something day" kommen?!

**Comment by NitramLegov** [comment]
Alles klar, wir achten darauf 👍

Falls am Samstag jemand die Kiste bauen möchte: wäre das eine Liste zum Handschleifen?  Beim schleifen mit den Maschinen ist es ja sinnvoller, den Staubsauger an die Maschine anzuschließen als an die Kiste.
Oder habe ich da einen Anwendungsfall vergessen?

**Comment by LukasFreeze** [comment]
Genau, das wäre eher für kleine/unförmige Teile und Kanten, die mit den Maschinen nicht geschliffen werden können :-)

PS: hatte das eigtl gestern schon einmal per Mail geschrieben aber sehe es jetzt nicht hier im Thread, ich hoffe da ist nix kaputt?!

**Comment by LukasFreeze** [comment]
Genau, das wäre eher für kleine/unförmige Teile und Kanten, die mit den
Maschinen nicht geschliffen werden können :-)
